Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -824,6 +826,19 @@
 		<ECProperty propertyName="TargetVolume" typeName="double" category="LayoutPondArea_Pond" kindOfQuantity="rru:VOLUME" displayLabel="Target Volume" description="Target volume." />
     </ECEntityClass>
 
+    <ECEntityClass typeName="LayoutParcelingArea" modifier="Sealed" displayLabel="Layout Parceling Area" description="Layout parceling area.">
+        <BaseClass>LayoutArea</BaseClass>
+
+        <ECProperty propertyName="MinParcelSurface" typeName="double" kindOfQuantity="rru:AREA" category="LayoutParcelingArea_ParcelLayout" displayLabel="Min Parcel Surface" description="Minimum parcel surface area." />
+        <ECProperty propertyName="MaxParcelDepth" typeName="double" kindOfQuantity="rru:LENGTH" category="LayoutParcelingArea_ParcelLayout" displayLabel="Max Parcel Depth" description="Maximum parcel depth." />
+        <ECProperty propertyName="MinParcelFrontage" typeName="double" kindOfQuantity="rru:LENGTH" category="LayoutParcelingArea_ParcelLayout" displayLabel="Min Parcel Frontage" description="Minimum parcel frontage." />
+        <ECProperty propertyName="parcelFrontageOffset" typeName="double" kindOfQuantity="rru:LENGTH" category="LayoutParcelingArea_ParcelLayout" displayLabel="Frontage Offset" description="Frontage offset." />
+
+        <ECProperty propertyName="HasBuilding" typeName="boolean" category="LayoutParcelingArea_BuildingLayout" displayLabel="Has Building" description="Indicates if the parcel has a building." />
```

Comment:
`HasBuilding` looks like it should be an Aspect pattern - same as we did for "HasParking", "HasSidewalk", etc. properties.